src/nvim/testdir/test_timers.vim

```diff
@@ -314,4 +313,30 @@ func Test_restore_count()
   call delete('Xtrctext')
 endfunc
 
+" Test that the garbage collector isn't triggered if a timer callback invokes
+" vgetc().
+func Test_nocatch_garbage_collect()
+  " 'uptimetime. must be bigger than the timer timeout
+  set ut=200
+  call test_garbagecollect_soon()
+  " skipped: Nvim does not support test_override()
+  return
```

Comment:
`throw` won't work here because of `_nocatch_` in function name.

Reply:
we might as well skip `test_garbagecollect_soon` then.